core/arch/aarch64/instr_create.h

```diff
@@ -65,41 +65,30 @@
  */
 #define ISZ_DOUBLE 3
 
-/**
- * Used in an additional immediate source operand to a vector operation, denotes
- * half-precision floating point vector elements. See \ref sec_IR_AArch64.
- */
-#define FSZ_HALF 1
-
 /**
  * Operand indicating half-precision floating point vector elements for the
  * other operands of the containing instruction.
  */
-#define OPND_CREATE_HALF() OPND_CREATE_INT8(FSZ_HALF)
+#define OPND_CREATE_BYTE() OPND_CREATE_INT8(ISZ_BYTE)
 
-/**
- * Used in an additional immediate source operand to a vector operation, denotes
- * single-precision floating point vector elements. See \ref sec_IR_AArch64.
- */
-#define FSZ_SINGLE 2
 
 /**
- * Operand indicating single-precision floating point vector elements for the
+ * Operand indicating half-precision floating point vector elements for the
  * other operands of the containing instruction.
  */
-#define OPND_CREATE_SINGLE() OPND_CREATE_INT8(FSZ_SINGLE)
+#define OPND_CREATE_HALF() OPND_CREATE_INT8(ISZ_HALF)
 
 /**
- * Used in an additional immediate source operand to a vector operation, denotes
- * double-precision floating point vector elements. See \ref sec_IR_AArch64.
+ * Operand indicating single-precision floating point vector elements for the
+ * other operands of the containing instruction.
  */
-#define FSZ_DOUBLE 3
+#define OPND_CREATE_SINGLE() OPND_CREATE_INT8(ISZ_SINGLE)
 
 /**
- * Operand indicating double-precision floating point vector elements for the
- * other operands of the containing instruction.
+ * Operand indicating 64 bit wide vector elements for the other operands of the
```

Comment:
I'm confused about removing floating-point from this comment yet leaving the name _DOUBLE.

Reply:
The assembly syntax for NEON uses the same arrangement specifiers indicating element width for floating point and integer instructions: b,h,s,d. IIUC s stands for (single) word and d for doubleword in this context.